### PR TITLE
Clarify some of the template creation steps

### PR DESCRIPTION
**Pull request by jorisw**
### What does it do?

Clarifies some of the steps for creating a template.

### Why is it needed?

Without the changes, I found some of the steps ambiguous.

### Related issue(s)/PR(s)

None

### CODE REVIEW

File: docs/v3.x/concepts/templates.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ After reading the above rules, follow these steps to create your template:
...
 2. Customize your app to match the needs of your use case.
-3. Outside of Strapi, create a new directory for your template.
-4. Create a `template.json` file in your template directory.
-5. If you have modified your app's `package.json`, include these changes (and _only_ these changes) in `template.json` in a `package` property. Otherwise, leave it as an empty object.
-6. Create a `/template` subdirectory.
+3. Outside of your Strapi app's directory, create a new directory for your template.
+4. Create a `template.json` file in your template's directory.
+5. If you have modified your app's `package.json`, include these changes (and _only_ these changes) in `template.json`, in a `package` property. Otherwise, have the template file represent an empty object, i.e. `{}`.
+6. Create a `template/` subdirectory.
 7. Think of all the files you have modified in your app, and copy them to the `/template` directory
```
Comment (remidej) [inline docs/v3.x/concepts/templates.md:98]: Maybe replace `/template` with `template` for consistency

```suggestion
7. Think of all the files you have modified in your app, and copy them to the `template` directory
```

File: docs/v3.x/concepts/templates.md [hunk 2]
```diff
@@ -91,9 +91,9 @@ After reading the above rules, follow these steps to create your template:
 
 1. Create a standard Strapi app with `create-strapi-app`, using the `--quickstart` option.
 2. Customize your app to match the needs of your use case.
-3. Outside of Strapi, create a new directory for your template.
-4. Create a `template.json` file in your template directory.
-5. If you have modified your app's `package.json`, include these changes (and _only_ these changes) in `template.json` in a `package` property. Otherwise, leave it as an empty object.
-6. Create a `/template` subdirectory.
+3. Outside of your Strapi app's directory, create a new directory for your template.
+4. Create a `template.json` file in your template's directory.
+5. If you have modified your app's `package.json`, include these changes (and _only_ these changes) in `template.json`, in a `package` property. Otherwise, have the template file represent an empty object, i.e. `{}`.
```
Comment (remidej) [inline docs/v3.x/concepts/templates.md:96]: > have the template file represent 

I think there should be a better way to phrase this. As a non-native, it seems odd to me

**Review by remidej — CHANGES_REQUESTED** [review]
Thanks for the suggestions, we're always happy to make the docs clearer. Please just review my 2 comments so we can merge

**Comment by remidej** [comment]
Thanks! Now @jorisw you just need to sign your commits so we can merge. See [the instructions on the DCO check](https://github.com/strapi/strapi/pull/8482/checks?check_run_id=1319567242)

**Review by remidej — APPROVED** [review]
Hello @jorisw

Good news: we don't need you to sign your commits anymore to merge this. Instead, you can click the [link](https://cla.strapi.io/strapi/strapi?pullRequest=8482) shared by the Strapi CLA bot, and follow the process there.

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Pull request merged**